drivers/bridge/bridge.go

```diff
@@ -145,6 +146,159 @@ func (c *NetworkConfiguration) Validate() error {
 	return nil
 }
 
+// FromMap retrieve the configuration data from the map form.
+func (c *NetworkConfiguration) FromMap(data map[string]interface{}) error {
+	if i, ok := data["BridgeName"]; ok {
```

Comment:
in addition to checking for ok, i think you should also check for `i != nil`.
This is because, you have assumed that the client will use a `map[string]string` to populate the data, but if the client decides to populate it via say json marshalling, then by default all the fields are marshalled with string values marked as nil.
Hence, the map lookup will succeed with ok = true, but the value (i) will be nil and the subsequent type assertion will fail at runtime.